src/controllers/update/company.profile.controller.ts

```diff
@@ -0,0 +1,48 @@
+import { Session } from "@companieshouse/node-session-handler";
+import { NextFunction, Request, Response } from "express";
+import { ApplicationData } from "model";
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+import { getApplicationData, mapOverseasEntityToDTO, setExtraData } from "../../utils/application.data";
+import { getCompanyRequest } from "../../service/overseas.entities.service";
+
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.CONFIRM_OVERSEA_ENTITY_DETAILS_PAGE}`);
+    const session = req.session as Session;
+    const appData: ApplicationData = getApplicationData(session);
+    const id: string = appData?.oe_number || "";
+    const companyDataResponse = await getCompanyRequest(req, id);
+    if (!companyDataResponse){
+      return res.redirect(config.OVERSEAS_ENTITY_QUERY_URL);
+    }
+
+    const companyData = mapOverseasEntityToDTO(companyDataResponse);
```

Comment:
Is this not the other way round? mapCompanyProfileToOverseasEntity + name of local var comanyData.